stock_picking_batch_packaging_qty/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
```

Comment:
suggestion: please always use https://gitlab.com/moduon/odoo-module-template when creating new modules, so all this boilerplate is correct.

```suggestion
# Copyright 2024 Moduon Team S.L.
# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
```